modules/consensus/accept_test.go

```diff
@@ -321,13 +321,13 @@ func TestInconsistentCheck(t *testing.T) {
 	// Corrupt the consensus set.
 	var sfod types.SiafundOutputID
 	var sfo types.SiafundOutput
-	for id, output := range cst.cs.siafundOutputs {
+	cst.cs.db.forEachSiafundOutputs(func(id types.SiafundOutputID, output types.SiafundOutput) {
 		sfod = id
 		sfo = output
-		break
-	}
+	})
```

Comment:
I'm considering some sort of cursor based thing. What I'm apprehensive about are the couple of spots where you need an arbitrary element that fits certain conditions. An implementation of something like firstSiacoinOutputs(func conditions(k, v) bool) would be just as complicated as the for each loop. Getting a single arbitrary value won't require passing in a function and should be much simpler. But even this only has a couple use cases inside of tests.